submissions/zhenyakornilov/html-css-popup/styles.css

```diff
@@ -0,0 +1,260 @@
+* {
+  margin: 0;
+  margin-bottom: 0;
+  box-sizing: border-box;
+  outline-style: none;
+}
+
+body {
+  font-family: "Montserrat", sans-serif;
+  background-color: #babfd1;
+}
+
+.header {
+  background-color: #ffadc2;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+}
+
+.nav-logo-text {
+  font-size: 20px;
+  font-weight: 700;
+  margin-left: 10px;
+}
+
+.nav-logo-text,
+.link {
+  color: #607196;
+}
+
+.nav-links {
+  display: flex;
+  justify-content: space-around;
+  line-height: 3em;
+  width: 30%;
+}
+
+.nav-item {
+  display: flex;
+  list-style: none;
+}
+
+.link {
+  text-decoration: none;
+  font-size: 1.1em;
+}
+
+.label-toggle {
+  display: flex;
+  cursor: pointer;
+  align-items: center;
+  justify-content: center;
+}
+
+.popup-toggle {
+  appearance: none;
+}
+
+.popup {
+  position: absolute;
+  right: 6em;
+  top: 4em;
+  width: 20em;
+  height: 26.5em;
+  background-color: #fff;
+  border: 1px solid #ffadc2;
+  box-shadow: 0 0 10px #ffadc2;
+  border-radius: 10px;
+  display: none;
+  overflow-y: auto;
+}
+
+.popup-wrap {
+  display: contents;
+  list-style: none;
+}
+
+.popup::after {
+  content: "";
+  position: fixed;
+  width: 15px;
+  height: 15px;
+  transform: translate(50%) rotate(-45deg);
+  background-color: #fff;
+  border-top: 1px solid #ffadc2;
+  border-right: 1px solid #ffadc2;
+  box-shadow: 1px -1px 0.5px #ffadc2;
+  right: 7.4%;
```

Comment:
why value `right ` has % units and `top` em. Any reason for that?

Reply:
I used % units on right to move popup arrow more smoothly on resize.

Reply:
This is not optimal because you have to hardcode value in media queries and even it won't help in all possible widths. 
![image](https://user-images.githubusercontent.com/21179601/183762664-dd83674b-ac3f-495d-8229-c2a8a1bf81d0.png)